app/tailwind.config.js

```diff
@@ -570,6 +570,7 @@ module.exports = {
       8: '2rem',
       9: '2.25rem',
       10: '2.5rem',
+      'extra-loose': '3.75rem',
```

Comment:
Are you using it? I search for it but I didn't find any place where you were using it. Could you remove it?